lambdas/death-enrichment/src/main/java/uk/gov/di/data/lep/DeathEnrichment.java

```diff
@@ -42,9 +48,15 @@ public String handleRequest(SQSEvent sqsEvent, Context context) {
     }
 
     @Tracing
-    private DeathNotificationSet enrichData(GroJsonRecord baseData) {
-        logger.info("Enriching and mapping data (sourceId: {})", baseData.registrationID());
+    private DeathNotificationSet enrichData(GroJsonRecordWithCorrelationId baseData) {
+        logger.info("Enriching and mapping data (sourceId: {})", baseData.groJsonRecord().registrationID());
```

Comment:
I think given the comments about sensitivity it might be worth using the correlation id here instead of the registration ID

Reply:
Yeah, I was wondering this - so all logs we want correlation id?

Reply:
I think where we've got a correlation ID we should use it, and at at least one point we need to log the correlation id with a source ID so we can link the two in the rest of the audit trail

Reply:
Sure - do we think think that the DeathEnrichment logs now contain PII, if they just have the correlationID? As we've marked them as PII in the template currently